Makefile

```diff
@@ -26,6 +26,22 @@ securedrop-workstation-grsec: ## Builds Debian metapackage for Qubes Workstation
 install-deps: ## Install initial Debian packaging dependencies
 	./scripts/install-deps
 
+.PHONY: requirements
+requirements: ## Creates requirements files for the Python projects
+	./scripts/create-requirements
+	./scripts/update-requirements
+
+.PHONY: build-wheels
+build-wheels: syncwheels ## Builds the wheels and syncs to the localwheels directory
+	./scripts/build-sync-wheels -p ${PKG_DIR}
+	./scripts/sync-sha256sums
+	./scripts/createdownloadurls.py > wheelsurls.txt
+
+.PHONY: sync-sha256sums
```

Comment:
Looks like we _don't_ need this makefile target? If it is not required, let's remove it, or document in what scenario it should be used.

Reply:
The dev-facing instructions in `scripts/update-requirements` recommend its use `if missing_wheels`; so looks like it should stay for now.

Reply:
isn't that `make build-wheels` not `make sync-sha256sums`?

Reply:
ok implemented